### PR TITLE
Namu Temple Ruins

### PR DESCRIPTION
**Pull request by vtange**
A pretty nice historical story here. Wonder what it could entail

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -7568,26 +7568,26 @@ QUEST_LV_0200_20150714_007571	I am not sure if you've noticed, but these rubber
...
-QUEST_LV_0200_20150714_007589	Since then, I've spent countless hours by pursuing the new role.{nl}But, the duty didn't give me the answer for my question.
-QUEST_LV_0200_20150714_007590	That's why I went to see Guiltine.{nl}After listening to my repetitive questions, she opened her mouth after a long silence.{nl}{nl}
-QUEST_LV_0200_20150714_007591	I am not going to give you the answer.{nl}But, I can give you the power.{nl}
-QUEST_LV_0200_20150714_007592	The power may lead you to the answer that you are seeking for.{nl}Even if you don't get the answer, you will be satisfied with new change that will be different from now.
-QUEST_LV_0200_20150714_007593	I moved out without saying anything.{nl}But, I realized after I found myself sobbing sadly.{nl}{nl}
-QUEST_LV_0200_20150714_007594	When the sun goes down tomorrow, I will go meet Guiltine and tell her my determination.
+QUEST_LV_0200_20150714_007575	$When you pull the levers in order, the tombstones may react.
+QUEST_LV_0200_20150714_007576	$You seem to have some other request to me.{nl}How can I help you?
```
Comment (TentuZero) [inline QUEST_LV_0200.tsv:7572]: Other request for me?

Comment (vtange) [inline QUEST_LV_0200.tsv:7572]: lol i read too fast

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -8456,60 +8456,60 @@ QUEST_LV_0200_20150714_008459	Collect %s by throwing red poison bottles to monst
...
-QUEST_LV_0200_20150714_008486	Return to the tombstone in Namu Temple Ruins
-QUEST_LV_0200_20150714_008487	You were able to obtain some food boxes which green rofdlems like from the blacksmith in Klaipeda. First, go back to Namu Temple Ruins.
-QUEST_LV_0200_20150714_008488	Look at the tombstone
-QUEST_LV_0200_20150714_008489	Return to the tombstone and check the status of it.
-QUEST_LV_0200_20150714_008490	Collect Sticky Resin
+QUEST_LV_0200_20150714_008480	$Return to the tombstone and try to restore it
+QUEST_LV_0200_20150714_008481	$It seems that you've collected all fragments of the tombstone. Return to the tombstone and take a look at it to see if you can restore it.
+QUEST_LV_0200_20150714_008482	$Collect %s from Minos
+QUEST_LV_0200_20150714_008483	$Ask the blacksmith in Klaipeda about adhesives
+QUEST_LV_0200_20150714_008484	$Ask the blacksmith in Klaipeda about adhesives about adhesives to restore the tombstone.
```
Comment (TentuZero) [inline QUEST_LV_0200.tsv:8480]: About adhesives about adhesives?

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -8765,8 +8765,8 @@ QUEST_LV_0200_20150717_008768	$Lastly, we need to collect Flame Sources.{nl}You
 QUEST_LV_0200_20150717_008769	I can feel strange energy.{nl}It seems that the status of the jewel of prominence is not well.
 QUEST_LV_0200_20150717_008770	If Antares used the power correctly, the Mage Tower would have not turned out like this.{nl}It is so unfortunate.
 QUEST_LV_0200_20150717_008771	Ah. There is diluting solution that I've requested to the Item Merchant in Fedimian.{nl}Can you also bring that?
-QUEST_LV_0200_20150717_008772	To glue the stones, we often use the sticky resin of Infrogalas Mages.{nl}But, you can't find it easily at Namu Temple Ruins.{nl}
-QUEST_LV_0200_20150717_008773	It will be easier to grab a star from the sky if you try to find it without using a food box.{nl}Otherwise, you should go to Viltis Forest.
+QUEST_LV_0200_20150717_008772	$To glue stone together, we often use the sticky resin of Infrogalas Mages.{nl}But, you can't find them easily at Namu Temple Ruins.{nl}
```
Comment (TentuZero) [inline QUEST_LV_0200.tsv]: the stone

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -7568,26 +7568,26 @@ QUEST_LV_0200_20150714_007571	I am not sure if you've noticed, but these rubber
...
+QUEST_LV_0200_20150714_007589	$Since then, I've spent countless hours performing my new role.{nl}However, my duties didn't give an answer to my questions.
+QUEST_LV_0200_20150714_007590	$That's why I went to see Giltine.{nl}After listening to my repetitive questions, she finally opened her mouth after a long silence.{nl}{nl}
+QUEST_LV_0200_20150714_007591	$I am not going to give you the answer.{nl}But, I can give you the power.{nl}
+QUEST_LV_0200_20150714_007592	$The power that may lead you to the answer that you are seeking.{nl}Even if you never get the answer, you will be satisfied with your new changes.
+QUEST_LV_0200_20150714_007593	$I immediately left without saying anything.{nl}But soon I...realizing I was so close to finally getting something...{nl}found myself sobbing...and realized what I truly wanted{nl}{nl}
+QUEST_LV_0200_20150714_007594	$When the sun goes down tomorrow, I will return to Giltine. And tell her my decision.
 QUEST_LV_0200_20150714_007595	Archeologist Yustas
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:7591]: Yustas -> Justas

(If you don't mind replacing)

Comment (vtange) [inline QUEST_LV_0200.tsv:7591]: I think Yustas isn't on this map so I avoided touching him for now for organization's sake

Comment (Sunaries) [inline QUEST_LV_0200.tsv:7591]: Alright, then I'll change it tomorrow or else I forget about it.

**Comment by vtange** [comment]
should be good to merge

**Comment by imcgames** [comment]
:+1:

**Pull request merged**